packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -116,6 +127,26 @@ export const ApiExplorer: FC<ApiExplorerProps> = ({
     return () => unregisterEnvAdaptor()
   }, [])
 
+  useEffect(() => {
+    // reconcile local storage state with URL or vice versa
```

Comment:
as we discussed, in the future this might not be an either or scenario. Adding this comment for other reviewers

Reply:
@josephaxisa Is there anything left to discuss right now here?

Reply:
I don't think so